po/pt_BR.po

```diff
@@ -73520,7 +73520,7 @@ msgid "Precede each output line with count of its occurrence"
 msgstr ""
 
 #: /tmp/fish/implicit/share/completions/uniq.fish:14
-msgid "Avoid comparing fist N characters"
+msgid "Avoid comparing first N characters"
```

Comment:
The tests are failing because this message id already exists. Simply remove this paragraph.